README.md

```diff
@@ -12,7 +12,8 @@ This web application is built to display covid-19 data of countries affected by
 
 ## Live Demo
 
-[View Life](https://duls-travelers-hub.herokuapp.com/)
+- [View video](https://www.loom.com/share/2f173a6ca1594c60a9e760d6d6ef74a3)
+- [View Demo](https://duls-covid-reporter.herokuapp.com/)
   
```

Comment:
- [ ] To improve the professionalism aspect of your ReadMe, kindly change the header title _Authors_ to _Author_, as this project is done solely.